public/static/app/perf/TestSample.js

```diff
@@ -19,28 +18,30 @@ mciModule.factory('TestSample', function() {
       return this._threads;
     }
 
-    this.testNames = function(){
+    this.testNames = function () {
       var tests = _.pluck(this.sample.data.results, "name");
       if (tests.length === 0) {
         tests = _.pluck(_.pluck(this.sample, "info"), "test_name");
       }
       return tests
     }
 
-    this.getLegendName = function(){
-      if(!!this.sample.tag){
+    this.getLegendName = function () {
+      if (!!this.sample.tag) {
         return this.sample.tag
       }
-      return this.sample.revision.substring(0,7)
+      return this.sample.revision.substring(0, 7)
     }
 
     // Returns only the keys that have results stored in them
-    this.resultKeys = function(testName){
+    this.resultKeys = function (testName) {
       var testInfo = this.resultForTest(testName);
-      return _.pluck(_(testInfo.results).pairs().filter(function(x){return typeof(x[1]) == "object"}), 0)
+      return _.pluck(_(testInfo.results).pairs().filter(function (x) {
+        return typeof (x[1]) == "object"
```

Comment:
Either version of this code will filter __undefined__ but not __null__:

```
typeof null
"object"
var x
testInfo = {results: {25: {AverageLatency: 3431223.721315061}, 30: null, 60: x, 120: 1}}
_.pluck(_(ttestInfo.results).pairs().filter(function (x) {return typeof (x[1]) == "object"}), 0)
["25", "30"]

// The following will do what I think you want:
_.chain(testInfo.results).pick(_.isObject).keys().value()
["25"]
```

Would it make sense to simplify it to the following:
```
return _.chain(testInfo.results).pick(_.isObject).keys().value()
```

Reply:
Sorry, this got changed by my formatter. I'm hesitant to change it at this time